### PR TITLE
Fix links in some pages (2)

### PR DESCRIPTION
**Pull request by tengqm**
This PR fixes some links that are using redirect records or dangling.

### CODE REVIEW

File: content/en/docs/contribute/participate/roles-and-responsibilities.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Anyone can:
 - Suggest improvements on [Slack](https://slack.k8s.io/) or the
   [SIG docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
 
-After [signing the CLA](/docs/contribute/new-content/overview/#sign-the-cla), anyone can also:
+After [signing the CLA](https://github.com/kubernetes/community/blob/master/CLA.md), anyone can also:
```
Comment (Sea-n) [inline content/en/docs/contribute/participate/roles-and-responsibilities.md:35]: I think the `git.k8s.io` short link would be better.

```suggestion
After [signing the CLA](https://git.k8s.io/community/CLA.md), anyone can also:
```

Comment (tengqm) [reply content/en/docs/contribute/participate/roles-and-responsibilities.md:35]: Short links are good for someone, but they may not **work** for everyone.

**Comment by reylejano** [comment]
Thank you for the updates
/approve

**Pull request merged**